src/index.html

```diff
@@ -0,0 +1 @@
+<pug src="./index.pug" />
```

Comment:
As for me it is better to move some parts from `index.pug` to `index.html` (like head, body, and other important parts of HTML). I think this will improve readability

Like [here](https://github.com/logux/logux.io/blob/main/src/index.html)